src/utils/cpg.ts

```diff
@@ -77,6 +77,23 @@ export class CPGExporter {
     this.libraries = this.createLibraries();
   }
 
+  modelExport(): Pathway {
```

Comment:
Two comments about this:
1) Everything else in this file is specific to CPG. It seems a little weird to create a new `CPGExporter` instance and then export to something other than CPG.
2) There was already the `exportPathway` method in `utils/builder.tsx` which can be modified

Reply:
I was just working in this class for now because I was using some of its features (e.g. it gathers up all the criteria libraries).  Ultimately I think this class would have become more general or it would have moved.  I'll take a look at the version in `utils/builder.tsx` though...didn't realize that was there 👀 .

Reply:
I'll keep this in mind though and rework it.

Reply:
I updated this so there's a new exporter class for the Camino model. Eventually I think we should extract the logic into 3 classes - CPGExporter, CaminoExporter, and common, but I'd rather wait until after Keeyan's PR is merged to avoid a potentially painful rebase.